project/plugins.sbt

```diff
@@ -1,16 +1,12 @@
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
-resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(
+  Resolver.ivyStylePatterns
+)
 
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.8.0")
```

Comment:
I can see this file is not formatted as well

Reply:
done